### PR TITLE
Spanish translations

### PR DESCRIPTION
**Pull request by parinisantiago**

### CODE REVIEW

File: locale/es_ES/submission.xml [hunk 1]
```diff
@@ -781,4 +779,18 @@
...
+	<message key="submission.authorWithAffiliation">{$name}, {$affiliation}</message>
+	<message key="submission.event.participantAdded">se asignó {$name}  a este envío como {$userGroupName}.</message>
+	<message key="submission.howToCite">Como citar</message>
+	<message key="submission.howToCite.citationFormats">Formatos de citas</message>
+	<message key="submission.submit.submissionLocaleDescription">Se aceptan envíos en varios idiomas. Elija el idioma principal del envío desde el menú desplegable de abajo.</message>
+	<message key="submission.upload.proof">Subir un archivo listo para publicación</message>
+	<message key="notification.type.copyeditorRequest">Se le ha pedido que revise las correcciones para "{$title}".</message>
+	<message key="notification.type.layouteditorRequest">Se le ha pedido revisar los diseños de "{$title}".</message>
+	<message key="notification.type.indexRequest">Se le ha pedido que cree un índice para "{$title}".</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/es_ES/submission.xml]: empty keys

File: locale/es_ES/submission.xml [hunk 2]
```diff
@@ -771,8 +769,8 @@
 	<message key="submission.query.closed">Cerrado</message>
 	<message key="submission.query.addNote">Añade mensaje</message>
 	<message key="grid.action.openNoteForm">Añade una respuesta nueva a la discusión</message>
-	<message key="submission.query.activity">Hay actividad nueva en la conversación.</message>
-	<message key="submission.query.new">Hay una pregunta nueva pendiente de respuesta.</message>
+	<message key="submission.query.activity">{$responderName} respondió a {$noteTitle}: {$noteContents}</message>
```
Comment (mtub) [inline locale/es_ES/submission.xml]: ok

File: locale/es_ES/user.xml [hunk 3]
```diff
@@ -146,7 +146,14 @@
...
-        <message key="user.login.rememberUsernameAndPassword">Recordar Usuario</message>
+	<message key="user.register.form.passwordLengthRestriction">La contraseña debe tener como mínimo {$length} caracteres.</message>
+	<message key="user.login.registrationComplete">Registro completo</message>
+	<message key="user.login.registrationComplete.instructions">¡Gracias por registrarte! ¿Qué te gustaría hacer después?</message>
+	<message key="user.login.registrationComplete.manageSubmissions">Ver envíos</message>
+	<message key="user.login.registrationComplete.newSubmission">Realizar un nuevo envío</message>
+	<message key="user.login.registrationComplete.continueBrowsing">Continuar navegando</message>
+	<message key="user.login.registrationPendingValidation">Registro pendiente de verificación</message>
+	<message key="user.login.rememberUsernameAndPassword">Manténme conectado</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/es_ES/user.xml]: empty key

**Comment by mtub** [comment]
Thanks @parinisantiago - and thanks @jordilacruz for checking this.

@parinisantiago , if you want to contribute to the Spanish translation in the future, maybe talk to Jordi and Marc (see https://pkp.sfu.ca/wiki/index.php?title=OJS:_Spanish_(es_ES)) about how they are planning to further work on the translation.

**Comment by marcbria** [comment]
Hi @mtub 

I think we messed it up a little:

@jordilacruz (that don't have write access) made comments to the pkp repo.
@parinisantiago saw them so decided to include those comments in his own repo (but still no PR).
So now you merged @parinisantiago files with @jordilacruz comments, but no real changes.

The good ones are those (thanks @parinisantiago to include them. Please PR to pkp)
https://github.com/parinisantiago/pkp-lib/commit/055c88f8f570c1823ef3bd928a3b3ac4622dbc72

Please @mtub, to avoid reproduce again messy workflow, I think the best solution is giving write access to the repository to @jordilacruz so he can review and change the lang files directly.

@mtub, what do you think?

**Comment by mtub** [comment]
Hi @marcbria , I think you misread what has happened:

@parinisantiago has updated their pull request - PR are updated if you send further commits to your repository. The correction commit has been part of the PR and is part of PKP's pkp-lib now:

https://github.com/pkp/pkp-lib/commit/055c88f8f570c1823ef3bd928a3b3ac4622dbc72

**Comment by marcbria** [comment]
You are write @mtub. I misread it all. ;-)

Anyway, don't you think it all will be easier if @jordilacruz could modify future PR instead of commenting and wait the initial translator to make the modification?

With direct changes @jordilacruz won't need to review stuff twice as he did now to be sure all the comments were applied.

**Pull request merged**